src/pages/api/github/dir.ts

```diff
@@ -0,0 +1,56 @@
+import { DirectoryRes } from "@/types/api/dir";
+import { Octokit } from "@octokit/core";
+import type { NextApiRequest, NextApiResponse } from "next";
+import { getSession } from "next-auth/react";
+
+async function getAllRepoFolder(
+  octokit: InstanceType<typeof Octokit>,
+  path?: string
+) {
+  const auth = await octokit.request("GET /user");
+  // Fork the repository
```

Comment:
Remove this comment. This doesn't fork the repo

Reply:
Sure